### DISCUSSION TITLE
python files without .py extension

### DISCUSSION THREAD

**Discussion by mutt13y** [super-linter/super-linter#3160] [Q&A]
Superlinter detects python files with no extension `uses: github/super-linter/slim@v4`
```
2022-07-19 14:08:43 [WARN]   Found Python script without extension:[.py]
2022-07-19 14:08:43 [INFO]   Please update file with proper extensions.
```

The file is then not passed to flake8

is there a way to have files without extensions included in the lint check ?

update:
found found https://github.com/github/super-linter/blob/main/lib/functions/detectFiles.sh#L271

`SUPPRESS_FILE_TYPE_WARN`

Although the file should still have been considered

Edit2:
This stops the alert but it still is not passed to flake8. I have `filename=*` in .flake8

**Comment by ferrarimarco** [comment]
What is the request here? :)

If it's an issue, please open a new bug or feature request :)
